### PR TITLE
multi-cluster 1.4 specifics

### PR DESCRIPTION
**Pull request by satya-dillikar**
- Updated SG groups for all clusters
- Added Iterate-cluster to View-cluster to see in TAP-GUI
- updated & tested with TAP 1.4.0.rc.18  (several workarounds included)

### CODE REVIEW

File: tap-setup-scripts/src/functions.sh [hunk 1]
```diff
@@ -112,7 +111,9 @@ function installTanzuCLI {
       --release-version=$TAP_VERSION \
       --product-file-id=$TANZUCLI_FILE_ID
 
-    tar xvf $DOWNLOADS/$TANZUCLI_FILE_NAME -C $TANZU_DIR
+    TANZUCLI_FILE_NAME=`ls $DOWNLOADS/tanzu-framework-linux-*`
```
Comment (satya-dillikar) [inline tap-setup-scripts/src/functions.sh:114]: the new tanzu-framework-linux file name is changed.

File: templates/aws-tap-entrypoint-existing-vpc.template.yaml [hunk 2]
```diff
@@ -2502,8 +2514,9 @@ Resources:
             echo "Installing pivnet CLI..."
             wget -O "$tap_dir/downloads/pivnet" "https://github.com/pivotal-cf/pivnet-cli/releases/download/v${PivNetVersion}/pivnet-linux-$(dpkg --print-architecture)-${PivNetVersion}"
             install -o $user -g $user -m 0755 "$tap_dir/downloads/pivnet" /usr/local/bin/pivnet
-            echo "Installing tanzu CLI and staging tanzu-cluster-essentials..."
+            echo "Installing Tanzu CLI and Staging Tanzu-cluster-essentials..."
             su - $user -c "$tap_dir/src/install-tools.sh"
+            sleep 120
```
Comment (satya-dillikar) [inline templates/aws-tap-entrypoint-existing-vpc.template.yaml:2519]: this sleep command will be replaced by another PR from Stephani

**Review by leclaist — APPROVED** [review]
